doc/htmldoc/models/index.rst

```diff
@@ -0,0 +1,173 @@
+.. _modelsmain:
+
+
+Model directory
+===============
+
+
+
+Find a model
+------------
+
+NEST has over 100 models, choose an option for finding the one you need!
+
+.. grid:: 1 1 2 2
+  :gutter: 1
+
+  .. grid-item::
+
+     .. grid:: 1 1 1 1
+         :gutter: 1
+
+         .. grid-item-card:: Model selector
+
+            * Filter models by tags with our :ref:`model_selector`
+
+         .. grid-item-card:: A-Z list
+
+            * Get the :ref:`full_list`
+
+  .. grid-item::
+
+     .. grid:: 1 1 1 1
+         :gutter: 1
+
+
+         .. grid-item-card:: Learn more about
+
+
+            * :ref:`neuron models <neurons_index>`
+
+            * :ref:`synapse models <synapse_index>`
+
+            * :ref:`device models <device_index>`
+
+            * :ref:`creating and customizing models with NESTML <nestml_ad>`
+
+.. _model_selector:
+
+Model selector
+--------------
+
+
+Select a tag to display corresponding models.
+
+By selecting multiple tags, you can refine your search to models that match all selected tags.
+
+.. raw:: html
+
+    <div id="tag-container">
+        <!-- Tags will be populated here by JavaScript -->
+    </div>
+    <h3>List of models based on your selection </h3>
+    <div id="model-list"></div>
+    <script src="../_static/js/filter_models.js"></script>
+
+|
+|
+
+----
+
+
+.. _full_list:
+
+Complete A-Z list of models
+----------------------------
+
+.. dropdown::  Show/Hide list
+   :animate: fade-in-slide-down
+   :color: info
+
+
+   **Neurons**
+
+   {% for items in tag_dict %}
+   {% if items.tag == "neuron" %}
+   {% for item in items.models | sort %}
+   * :doc:`{{ item | replace(".html", "") }}`
+   {% endfor %}
+   {% endif %}
+   {% endfor %}
+
+   **Synapses**
+
+   {% for items in tag_dict %}
+   {% if items.tag == "synapse" %}
+   {% for item in items.models | sort %}
+   * :doc:`{{ item | replace(".html", "") }}`
+   {% endfor %}
+   {% endif %}
+   {% endfor %}
+
+   **Devices**
+
+   {% for items in tag_dict %}
+   {% if items.tag == "device" %}
+   {% for item in items.models | sort %}
+   * :doc:`{{ item | replace(".html", "") }}`
+   {% endfor %}
+   {% endif %}
+   {% endfor %}
+
+
+
+
+
+----
+
+What we mean by `models`
```

Comment:
Perhaps point out that the models on this page belong to the first category.

Reply:
done